### ISSUE TITLE
Provider is underpaid by a B2B client

### ISSUE THREAD

**Issue by adinetech** [mysteriumnetwork/node#5610]
**Describe the bug**
A provider is underpaid by a B2B client

**Expected behavior**
The MYST for the data transferred should be paid.

**Screenshots**
![image](https://user-images.githubusercontent.com/68943180/201336004-97990e4e-9e98-419d-9eb2-b2723e84a908.png)

**Calculations**
The B2B client consumed 519.78 GB and earned just 17.98383 MYST, but he should have earned ~38 MYST (based on the initial price)

**Required Information**
Provider identity: 0x1b8ec8d0ff4f9db819404f70587a6eba910c329c
Session identity: 52e2604e-96d0-483a-b25d-f99eb55a6f0f

**Additional context**
As it’s a B2B session, he (the provider) wants the Mysterium team to reimburse the underpaid MYST and fix this bug as soon as possible.

**Comment by etherunit** [comment]
@adinetech Could you request node logs? This could help to verify node issues.

**Comment by etherunit** [comment]
Based on our Slack discussion, it's hard to say what's the actual cause for this.

**What we know?**

Provider had the same data as consumer for the session and then it instantly doubled and session was disconnected at that moment.
There might be 2 cases:
- consumer disconnected the session because provider started to ask for overchage;
- consumer disconnected (lost connection) but provider didn’t know about that and were sending data after consumer disconnect.
- 
So, in short, consumer didn’t consume that traffic, that is why he didn’t pay for it.

And finally it could be 2 reasons:
- provider tried to cheat, but failed.
- some bug with session accounting or metrics in the node.

**Comment by etherunit** [comment]
@soffokl Is there anything else we can do about it?

**Comment by soffokl** [comment]
This happened because of a bug in metrics, the session traffic was counted twice by the provider. There was no real traffic passed through the provider. 
Not it's fixed and should show the correct number of session traffic even for historical information.
